src/calendar/view/CalendarAgendaView.ts

```diff
@@ -184,6 +186,13 @@ export class CalendarAgendaView implements Component<CalendarAgendaViewAttrs> {
 				icon: BootIcons.Calendar,
 				message: "noEntries_msg",
 				color: theme.list_message_bg,
+				bottomContent: renderNewEventButton({
+					label: "newEvent_action",
+					click: () => attrs.onCreateEvent?.(),
+					style: {
+						"margin-top": ".6rem",
```

Comment:
We have some utility classes for sizing and margins that we should use to keep the spacing standardized, take a look at [main-styles.ts#L266](https://github.com/tutao/tutanota/blob/971c09f66b0055e5b1af7684db53b6265c4c5ab0/src/gui/main-styles.ts#L266)